docs/mongocli/command/mongocli-ops-manager-serverUsage-download.txt

```diff
@@ -54,6 +54,10 @@ Options
      - string
      - false
      - Optional output file name. Uses the log name if the output file's name is not specified.
+   * - --skipRedaction
+     - 
+     - false
+     - If set to true, emails, hostnames, IP addresses, and namespaces in API responses involving this job are kept with their original values.
```

Comment:
Changing to fit our new format for boolean flags. I assume that "are kept" means Ops Manager includes them in logs, but please adjust if that's incorrect.
```suggestion
     - Flag that indicates whether Ops Manager skips redaction of sensitive values. If set to true, Ops Manager logs the original values for email addresses, hostnames, IP addresses, and namespaces in API responses for this job.
```